src/gobapi/auth/auth_query.py

```diff
@@ -81,6 +81,20 @@ def get_suppressed_columns(self):
             self._suppressed_columns = cols
         return self._suppressed_columns
 
+    def _is_secure_type(self, spec):
+        """
+        Tells if spec is a secure type
+        Either a plain secure type or a JSON that contains a secure type
+
+        :param spec:
+        :return:
+        """
+        gob_type = get_gob_type_from_info(spec)
+        if issubclass(gob_type, gob_secure_types.Secure):
+            return True
+        elif issubclass(get_gob_type_from_info(spec), gob_types.JSON):
```

Comment:
Why call get_gob_type_from_info(spec) again? ;)

Reply:
will fix this!